test/src/TestFlightResume.cpp

```
@@ -0,0 +1,378 @@
+    .active_taskpoint_index = harness.task_manager.GetActiveTaskPointIndex(),
+    /* rounded to the metre: the two passes must agree exactly, but the
+       assertion should not be hostage to a floating-point last bit */
+    .travelled_metres = (int)stats.total.travelled.GetDistance(),
+    .flight_time_seconds = (int)calculated.flight.flight_time.count(),
+    .altitude_samples =
+      (int)harness.computer.GetFlightStats().altitude.GetCount(),
+  };
+}
+
+static void
+Print(const char *label, const Outcome &o) noexcept
```

Cppcheck (2.21.0)

[error] Dereference of an invalid iterator

(derefInvalidIterator)